ConsentViewController/Classes/Consents/ConsentStatus.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ConsentStatus.swift
+//  Pods
+//
+//  Created by Andre Herculano on 31.08.22.
+//
+
+import Foundation
+
+struct ConsentStatus: Decodable, Equatable {
+    struct GranularStatus: Decodable, Equatable {
+        enum Status: String, Decodable, Equatable {
```

Comment:
nice, I'll use an enum also

Reply:
@carmelo-iriti I wanted to ask Sid if those are the only values we can receive, but I believe they are.